eng/common/scripts/Package-Properties.ps1

```diff
@@ -78,41 +78,27 @@ function Get-PkgProperties
     (
         [Parameter(Mandatory = $true)]
         [string]$PackageName,
-        [Parameter(Mandatory = $true)]
         [string]$ServiceDirectory
     )
 
-    $pkgDirectoryPath = $null
-    $serviceDirectoryPath = Join-Path $RepoRoot "sdk" $ServiceDirectory
-    if (!(Test-Path $serviceDirectoryPath))
+    if ($ServiceDirectory)
```

Comment:
Do we even need this condition? We can always blindly pass the ServiceDirectory parameter.